### PR TITLE
DDF-2882 added user interfaces docs to jbake templates.

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Adds `Using` sections (User interface guides) to the jbake-templated documentation. 

#### Who is reviewing it?

@vinamartin @Lambeaux @jrnorth @ahoffer 

#### Choose 2 committers to review/merge the PR.

@andrewkfiedler
@clockard
@coyotesqrl
@lessarderic
@pklinef
@shaundmorris

#### How should this be tested? (List steps with links to updated documentation)

- review attached pdf to ensure all sections from current documentation are included.
"New" `-contents.adoc` files are existing content with headers added. **Content Review is OPTIONAL**

*The Standard Search UI is imported from another module and won't appear in the "draft" form of the documentation, but will be included when the draft version replaces the non-template version.*

#### What are the relevant tickets?

[DDF-2882](https://codice.atlassian.net/browse/DDF-2882)

#### Screenshots (if appropriate)

[jdocumentation.pdf](https://github.com/codice/ddf/files/1072941/jdocumentation.pdf)

#### Checklist:
- [x] Documentation Updated
- [ ] Change Log Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: distribution/docs/src/main/jdocs/content/_using/using-catalog-search-ui-contents.adoc [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+.. Select a previous version from the list to preview.
+.. If desired, select *Revert to Selected Version* to undo changes made after that revision.
+
+==== Viewing Metadata Quality
+
+View and fix issues with metadata quality in a record.
+
+[NOTE]
+====
+Correcting metadata issues may require heightened permissions.
```
Comment (millerw8) [inline distribution/docs/src/main/jdocs/content/_using/using-catalog-search-ui-contents.adoc]: `heightened`?  `Elevated` maybe?  or `Special`, or `write`

Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_using/using-catalog-search-ui-contents.adoc]: Of those, I like `elevated` best. Maybe `administrative privileges` is more familiar to the audience?

File: distribution/docs/src/main/jdocs/content/_using/using-intro-contents.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+:title: Using Intro
+:type: usingIntro
+:status: published
+:summary: Introduction to User Interfaces.
```
Comment (clockard) [inline distribution/docs/src/main/jdocs/content/_using/using-intro-contents.adoc]: Should `User Interfaces` be capitalized?

Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_using/using-intro-contents.adoc]: probably not

Comment (ricklarsen) [inline distribution/docs/src/main/jdocs/content/_using/using-intro-contents.adoc]: fixed.

**Comment by ricklarsen** [comment]
build now

**Comment by ricklarsen** [comment]
build now

**Pull request merged**